doc/source/tutorials/qPDF/qPDF.ipynb

```diff
@@ -0,0 +1 @@
+../../../../examples/qPDF/qPDF.ipynb
```

Comment:
I noticed that some equations in the notebook are images and some are latex equations. It would be great to update this at some point so that everything is the same. I am not sure what appears better in the sphinx docs.